docs-partials/builder/vsphere/iso/CleanupConfig-not-required.mdx

```diff
@@ -0,0 +1,5 @@
+<!-- Code generated from the comments of the CleanupConfig struct in builder/vsphere/iso/step_cleanup.go; DO NOT EDIT MANUALLY -->
+
+- `destroy` (bool) - If set to true, the VM will be destroyed after the builder completes
```

Comment:
This field is not currently included in the documentation because the new partial file is not being included [here](https://github.com/hashicorp/packer-plugin-vsphere/blob/main/docs/builders/vsphere-iso.mdx?plain=1#L186). That said, by moving the destroy option into the CreateConfig structure, which is already included, the new argument will be displayed in the final documentation.